src/components/List.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const List = ({items}) => {
+    let listArray = items.map((item) => {
+        if (item === 'dog'){
+            return <li className='border border-gray-400 py-4 px-6' key={item}>{`🐶 ${item}`}</li>
+        }else if(item === 'cat'){
+            return <li className='border border-gray-400 py-4 px-6' key={item}>{`😸 ${item}`}</li>
+        }else if (item === 'chicken'){
+            return <li className='border border-gray-400 py-4 px-6' key={item}>{`🐔 ${item}`}</li>
+        }else if (item === 'cow'){
+            return <li className='border border-gray-400 py-4 px-6' key={item}>{`🐮 ${item}`}</li>
+        }else if (item === 'sheep'){
+            return <li className='border border-gray-400 py-4 px-6' key={item}>{`🐏 ${item}`}</li>
+        }
+        return <li className='border border-gray-400 py-4 px-6' key={item}>{`🐎 ${item}`}</li>
+    })
```

Comment:
It is cool, but it is somehow bad. Instead of doing this, you can create an object that contains the item's name and emoji, then loop through it to display different items. Also remember to use const on the variables that will not change.
```suggestion
const items = [
  { name: "dog", emoji: "🐶",id:1 },
  { name: "cat", emoji: "😸",id:2 },
  { name: "cow", emoji: "🐮",id:3 },
];


const listArray = items.map((item) => {
  return (
    <li className="border border-gray-400 py-4 px-6" key={item.id}>
      {item.emoji} {item.name}
    </li>
  );
})
```